models/note.go

```diff
@@ -0,0 +1,34 @@
+// Package models defines the data types, payloads and repository interfaces
+// of the accounts service.
+package models
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+)
+
+type NoteWithBlocks struct {
+	ID       uuid.UUID `json:"id" bson:"_id,omitempty"`
+	AuthorId string    `json:"authorId" bson:"authorId,omitempty"`
+	Title    string    `json:"title" bson:"title,omitempty"`
+	Blocks   []Block   `json:"blocks" bson:"blocks,omitempty"`
+}
```

Comment:
Is there such a thing as a note without blocks? What is the purpose of this type?